src/main/java/uk/gov/companieshouse/overseasentitiesapi/service/update/PopulateUpdateSubmission.java

```diff
@@ -0,0 +1,96 @@
+  public UpdateSubmission populate() {
+    // updateSubmission.type(); Type is already set in UserSubmission, has a default value of
+    // "OE02";
+    updateSubmission.setUserSubmission(this.overseasEntitySubmissionDto);
+    populateDueDiligence(this.overseasEntitySubmissionDto, updateSubmission);
+    populatePresenter(this.overseasEntitySubmissionDto, this.updateSubmission);
+    updateSubmission.setBeneficialOwnerStatement(
+        this.overseasEntitySubmissionDto.getBeneficialOwnersStatement().getValue());
+
+    updateSubmission.setAnyBOsOrMOsAddedOrCeased(
+        this.overseasEntitySubmissionDto.getUpdate().isRegistrableBeneficialOwner());
+    // AC contact details is covered in --> overseasEntitySubmissionDto.getPresenter().getEmail();
+    // TBD: NoChangesInFilingPeriodStatement will be done in
+    // https://companieshouse.atlassian.net/browse/UAR-461//
+    // TBD updateSubmission.setNoChangesInFilingPeriodStatement(); //
+
+    return updateSubmission;
+  }
+
+  public void populateDueDiligence(
+      OverseasEntitySubmissionDto overseasEntitySubmissionDto, UpdateSubmission updateSubmission) {
+    DueDiligenceDto dueDiligenceDto =
+        overseasEntitySubmissionDto.getDueDiligence(); // TBD .. which DueDiligence to use??//
+    DueDiligence dueDiligence = new DueDiligence();
+    dueDiligence.setDueDiligenceCorrespondenceAddress(dueDiligenceDto.getAddress());
+    dueDiligence.setAgentName(dueDiligenceDto.getName());
+    dueDiligence.setEmail(dueDiligenceDto.getEmail());
+    dueDiligence.setDateChecked(
+        dueDiligenceDto.getIdentityDate().toString()); // TBD .. check date format//
+    dueDiligence.setPartnerName(dueDiligenceDto.getPartnerName());
+    dueDiligence.setAgentAssuranceCode(dueDiligenceDto.getAgentCode());
+    dueDiligence.setSupervisoryBody(dueDiligenceDto.getSupervisoryName());
+    dueDiligence.setAmlRegistrationNumber(dueDiligenceDto.getAmlNumber());
+    updateSubmission.setDueDiligence(dueDiligence);
+  }
+
+  public void populateFilingForDate(
+      OverseasEntitySubmissionDto overseasEntitySubmissionDto, UpdateSubmission updateSubmission) {
+    FilingForDate filingForDate = new FilingForDate();
+    var filingDate = overseasEntitySubmissionDto.getUpdate().getFilingDate();
+    filingForDate.setDay((String.valueOf(filingDate.getDayOfMonth())));
+    filingForDate.setMonth((String.valueOf(filingDate.getMonth())));
+    filingForDate.setYear((String.valueOf(filingDate.getYear())));
+    updateSubmission.setFilingForDate(filingForDate);
+  }
+
+  public void populatePresenter(
+      OverseasEntitySubmissionDto overseasEntitySubmissionDto, UpdateSubmission updateSubmission) {
+    Presenter presenter = new Presenter();
+    presenter.setEmail(overseasEntitySubmissionDto.getPresenter().getEmail());
+    presenter.setName(overseasEntitySubmissionDto.getPresenter().getFullName());
+    updateSubmission.setPresenter(presenter);
+  }
+
+  public UpdateSubmission populate(
```

Comment:
why two public populate overloads? I think the one without args should be private? Could also pass the DTos in the constructor which might be better.